### PR TITLE
HAC-1808: Update release number in dynamic plugin sdk docs

### PR DESCRIPTION
**Pull request by TheRealJon**

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/lgtm cancel

Can we update the package versions in frontend/packages/console-dynamic-plugin-sdk/README.md as well?

File: frontend/packages/console-dynamic-plugin-sdk/README.md
Comment (jhadvig) [reply frontend/packages/console-dynamic-plugin-sdk/README.md:48]: @spadgett for the future reference, where do you got these version numbers from ?

**Comment by TheRealJon** [comment]
@spadgett I've committed your suggested changes. I'm curious though, won't future minor releases of the dynamic plugin packages still be compatible with 4.11 until we introduce a breaking change?

**Comment by spadgett** [comment]
Yeah, part of the problem is we're still using 0.y.z versions and anything <1.0.0 can have API breaking changes, so we have to document what specific versions are compatible. Even after 1.0.0, minor version updates can contain new APIs that won't be in the older OpenShift console versions, so only patch releases would be safe. After plugins are GA, we can document OpenShift 4.12 supports SDK version 1.0.x, 4.13 supports 1.1.x, etc. Then we have patch versions to backport bug fixes.

If we need to backport a bug fix to the 4.11 SDK, we might have to push a version like 0.0.11-patch-1 or similar and document it in the README (assuming we've already released a 0.0.12 that's incompatible).

**Review by spadgett — APPROVED** [review]
/lgtm

File: frontend/packages/console-dynamic-plugin-sdk/README.md
Comment (spadgett) [inline frontend/packages/console-dynamic-plugin-sdk/README.md:47]: I believe we can use 0.0.12 for 4.11, which contains the `Timestamp` component.

```suggestion
| 4.11.x | `@openshift-console/dynamic-plugin-sdk` | 0.0.12 |
```

cc @rawagner 
/hold

Comment (rawagner) [reply frontend/packages/console-dynamic-plugin-sdk/README.md:47]: That is correct

**Comment by spadgett** [comment]
/hold cancel

Unfortunately there's a merge a conflict :/

**Comment by TheRealJon** [comment]
@spadgett Done. I also added that missing yarn command to the root package.json

**Comment by TheRealJon** [comment]
QE Approver
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

**Review by RickJWagner — APPROVED** [review]
Looks good to me.
Approved.

**Pull request merged**
